astropy/modeling/models.py

```diff
@@ -7,7 +7,7 @@
 from __future__ import (absolute_import, unicode_literals, division,
                         print_function)
 
-from .core import custom_model
+from .core import custom_model  # pylint: disable=W0611
```

Comment:
I don't really like this import here in the first place, but being that it was already there I guess it should stay for now.